packages/xo-vmdk-to-vhd/src/virtual-buffer.js

```diff
@@ -1,35 +1,52 @@
 'use strict'
 
-import { Slicer } from 'pipette'
-
 export class VirtualBuffer {
   constructor(readStream) {
-    this.slicer = new Slicer(readStream)
+    const _this = this
```

Comment:
Remove if no longer used.

Reply:
It's not necessary, because you are using `async` functions which “inherits” the `this` of the outer scope.